### DISCUSSION TITLE
Majority of models now have no sample output images - Need a way to flag

### DISCUSSION THREAD

**Discussion by jmattthew** [civitai/civitai#34] [Ideas]
If you browsing Civitai by "newest", the majority of new models now have no sample output images.  It looks like Civitai has imported all the models by the huggingface acccount, sd-concepts-library.  That account has added 800 textual inversions models, and almost none of them include any samples of output.  They only show training images.

A library of training images isn't what most Civitai visitors are looking for.  When you see a great looking image while browsing, then you click on it, it's a bummer to learn that it's only a training image.  That tells you nothing about what kind of output the embedding will produce.

I'd be happy to help flag them if the report button included an option for "Report as inaccurate" or something like that.

**Comment by JustMaier** [comment]
Great idea. That's definitely the challenge with the things in that library.

**Comment by JustMaier** [comment]
This has been resolved. We've regenerated images for all of the TIs from that library and removed TIs that didn't produce decent results or that didn't have enough user engagement.
